adoptopenjdk-api-v3-models/src/test/kotlin/net/adoptopenjdk/api/VersionParserTest.kt

```diff
@@ -1,18 +1,20 @@
 package net.adoptopenjdk.api
 
-import java.net.URLDecoder
-import java.nio.charset.Charset
 /* ktlint-disable no-wildcard-imports */
-import java.util.*
 /* ktlint-enable no-wildcard-imports */
-import java.util.stream.Stream
+import net.adoptopenjdk.api.v3.dataSources.models.Releases
 import net.adoptopenjdk.api.v3.models.VersionData
 import net.adoptopenjdk.api.v3.parser.VersionParser
 import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertNull
 import org.junit.jupiter.api.Assertions.assertTrue
 import org.junit.jupiter.api.DynamicTest
 import org.junit.jupiter.api.Test
 import org.junit.jupiter.api.TestFactory
+import java.net.URLDecoder
+import java.nio.charset.Charset
+import java.util.*
```

Comment:
prefer explicit exports

Reply:
we should really have some IDE or `editorconfig` for this rule, I've seen you mention it at least 10 times :-)

it looks like it's an option here: https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties

but we appear to disable it for some reason https://github.com/AdoptOpenJDK/openjdk-api-v3/blob/master/.editorconfig#L944

Reply:
we can fix that, issue is default intellij config is to wildcard import java.* and some other stuff, and I am not sure if that can be changed from editorconfig

Reply:
possibly `ij_java_imports_layout`